ssr/src/page/post_view/video_loader.rs

```diff
@@ -48,7 +50,7 @@ pub fn BgView(
                 style:background-color="rgb(0, 0, 0)"
                 style:background-image=move || format!("url({})", bg_url(uid()))
             ></div>
-            <Show when=move || { idx == 4 && !is_connected.get() && show_login_popup.get() }>
+            <Show when=move || { current_idx.get() != 0 && current_idx.get() % 4 == 0 && !is_connected.get() && show_login_popup.get() }>
```

Comment:
2 things to confirm from PM:
- Do we want to repeatedly show after every N scrolls
- Should N be 4